plutus-core/plutus-ir/src/PlutusIR/Subst.hs

```diff
@@ -0,0 +1,21 @@
+module PlutusIR.Subst
+    ( uniquesTerm
+    , uniquesType
+    ) where
+
+import qualified PlutusCore.Core.Type as PLC
+import qualified PlutusCore.Name      as PLC
+import           PlutusCore.Subst     (uniquesType)
+
+import           PlutusIR.Core
+
+import           Control.Lens
+import           Data.Set             as Set
+
+setOf :: Getting (Set a) s a -> s -> Set a
```

Comment:
Can't we just use this from lens? http://hackage.haskell.org/package/lens-4.18.1/docs/Data-Set-Lens.html#v:setOf